src/components/pages/Contacts/index.tsx

```diff
@@ -306,6 +364,16 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
                       </div>
                     ))}
                   </td>
+                  <td>
+                    <Button
+                      title={'send ' + contact.first_name + ' a message'}
+                      onClick={() => {
+                        setShowSendModal(true);
+                        setContactBeingSentMail(contact);
+                      }}>
+                      <img src={require('./mail.svg')} />
```

Comment:
Using require inline is not recommended
I'd import like any other asset

Reply:
changed!